### PR TITLE
About kost

### PR DESCRIPTION
**Pull request by KostBoh**
Add changes About Us section

### CODE REVIEW

File: src/css/about-us.css [hunk 1]
```diff
@@ -0,0 +1,203 @@
+.about-section {
+  /* text-align: center; */
```
Comment (asaulyak) [inline src/css/about-us.css:2]: remove commented out code

File: src/css/about-us.css [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+}
+
+.container {
```
Comment (asaulyak) [inline src/css/about-us.css:5]: don't override global classes

File: src/css/about-us.css [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+    .about-title-container {
+      max-width: 832px;
+      /* margin-bottom: 64px; */
+    }
+  }
+}
+
+
+.section-title {
+    color: #121417;
```
Comment (asaulyak) [inline src/css/about-us.css:56]: use var for color

File: src/css/about-us.css [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+    text-align: center;
+    font-family: Manrope;
```
Comment (asaulyak) [inline src/css/about-us.css:58]: remove font family

File: src/css/about-us.css [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+    font-size: 32px;
+    font-style: normal;
```
Comment (asaulyak) [inline src/css/about-us.css:60]: remove font-style

File: src/css/about-us.css [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+    letter-spacing: -0.02em;
+  }
+}
+
+
+.section-title-accent {
+  position: relative;
+}
+
+.section-title-accent::after {
```
Comment (asaulyak) [inline src/css/about-us.css:80]: use global class for accent title

File: src/css/about-us.css [hunk 7]
```diff
@@ -0,0 +1,203 @@
...
+    gap: 32px;
+    
+  }
+}
+
+
+.about-item::before {
+  content: "0"counter(item-counter);
+  counter-increment: item-counter;
+  color: #4DC274;
```
Comment (asaulyak) [inline src/css/about-us.css:137]: use var for colors

File: src/css/about-us.css [hunk 8]
```diff
@@ -0,0 +1,203 @@
...
+    border-radius: 16px;
+    background: #F3F5F6;
+    padding: 32px;
+    
+}
+
+
+
+
+.about-number {}
```
Comment (asaulyak) [inline src/css/about-us.css:164]: remove

File: src/css/about-us.css [hunk 9]
```diff
@@ -0,0 +1,203 @@
...
+
+.about-text {
+    width: 271px;
+    height: 48px;
+    flex-shrink: 0;
+    color: #121417;
+    font-family: Manrope;
```
Comment (asaulyak) [inline src/css/about-us.css:171]: remove

File: src/css/about-us.css [hunk 10]
```diff
@@ -0,0 +1,203 @@
...
+
+
+
+.about-number {}
+
+.about-text {
+    width: 271px;
+    height: 48px;
+    flex-shrink: 0;
+    color: #121417;
```
Comment (asaulyak) [inline src/css/about-us.css:170]: use var

File: src/css/about-us.css [hunk 11]
```diff
@@ -0,0 +1,203 @@
...
+    font-family: Manrope;
+    font-size: 20px;
+    font-style: normal;
```
Comment (asaulyak) [inline src/css/about-us.css:173]: remove

File: src/css/about-us.css [hunk 12]
```diff
@@ -0,0 +1,203 @@
...
+    letter-spacing: -0.02em;
+    padding-left: 32px;
+    
+}
+
+.about-emoji {
+  width: 335px;
+  height: 313px;
+  border-radius: 16px;
+  background: #4DC274;
```
Comment (asaulyak) [inline src/css/about-us.css:185]: var

File: src/css/about-us.css [hunk 13]
```diff
@@ -0,0 +1,203 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+@media only screen and (min-width: 768px) {
+  .about-emoji {
+    width: 704px;
```
Comment (asaulyak) [inline src/css/about-us.css:193]: these styles come from global container

File: src/partials/about-us.html [hunk 14]
```diff
@@ -1 +1,44 @@
-<section>about us</section>
+<section class="about-section" id="About">
```
Comment (asaulyak) [inline src/partials/about-us.html:1]: class="about-section section"

Comment (asaulyak) [inline src/partials/about-us.html:1]: id="about"

File: src/partials/about-us.html [hunk 15]
```diff
@@ -1 +1,44 @@
...
+  <div class="container">
+    <div class="about-title-container">
```
Comment (asaulyak) [inline src/partials/about-us.html:3]: remove extra div
<div class="container about-title-container">

File: src/partials/about-us.html [hunk 16]
```diff
@@ -1 +1,44 @@
...
+          <p class="about-text">Multicultural learning environment</p>
+        </li>
+        <li class="about-item item">
+          <!-- <p class="about-number">04</p> -->
+          <p class="about-text">Comprehensive language support services</p>
+        </li>
+        <div class="about-emoji">
+          <img
+            srcset="
+              ../img/about-us/memoji.png    1x,
```
Comment (asaulyak) [inline src/partials/about-us.html:33]: image sources should start with one dot

Comment (asaulyak) [reply src/partials/about-us.html:33]: add width and height to the images

File: src/partials/about-us.html [hunk 17]
```diff
@@ -1 +1,44 @@
...
+        </li>
+        <li class="about-item item">
+          <!-- <p class="about-number">03</p> -->
+          <p class="about-text">Multicultural learning environment</p>
+        </li>
+        <li class="about-item item">
+          <!-- <p class="about-number">04</p> -->
+          <p class="about-text">Comprehensive language support services</p>
+        </li>
+        <div class="about-emoji">
```
Comment (asaulyak) [inline src/partials/about-us.html:30]: div can't be inside ul

File: src/css/about-us.css [hunk 18]
```diff
@@ -0,0 +1,203 @@
...
+  width: calc(100% + 4px);
+  height: 20px;
+  border-radius: 8px;
+  background: var(--soft-green);
+}
+
+.about-section-text {
+color: rgba(18, 20, 23, 0.50);
+text-align: center;
+font-family: Manrope;
```
Comment (asaulyak) [inline src/css/about-us.css:95]: remove

File: src/css/about-us.css [hunk 19]
```diff
@@ -0,0 +1,203 @@
...
+font-size: 14px;
+font-style: normal;
```
Comment (asaulyak) [inline src/css/about-us.css:97]: remove

**Review by asaulyak — COMMENTED** [review]
remove all commented out code

File: src/css/about-us.css (hunk 1 above)
Comment (KostBoh) [reply src/css/about-us.css:2]: +

File: src/css/about-us.css (hunk 2 above)
Comment (KostBoh) [reply src/css/about-us.css:5]: +

File: src/css/about-us.css (hunk 3 above)
Comment (KostBoh) [reply src/css/about-us.css:56]: +

File: src/css/about-us.css (hunk 5 above)
Comment (KostBoh) [reply src/css/about-us.css:60]: +

File: src/css/about-us.css (hunk 7 above)
Comment (KostBoh) [reply src/css/about-us.css:137]: +

File: src/css/about-us.css (hunk 6 above)
Comment (KostBoh) [reply src/css/about-us.css:80]: +

File: src/css/about-us.css (hunk 8 above)
Comment (KostBoh) [reply src/css/about-us.css:164]: +

File: src/css/about-us.css (hunk 9 above)
Comment (KostBoh) [reply src/css/about-us.css:171]: +

File: src/css/about-us.css (hunk 10 above)
Comment (KostBoh) [reply src/css/about-us.css:170]: +

File: src/css/about-us.css (hunk 9 above)
Comment (KostBoh) [reply src/css/about-us.css:171]: I think only font-family

File: src/css/about-us.css (hunk 11 above)
Comment (KostBoh) [reply src/css/about-us.css:173]: dell only font-style?

File: src/css/about-us.css (hunk 12 above)
Comment (KostBoh) [reply src/css/about-us.css:185]: +

File: src/partials/about-us.html (hunk 14 above)
Comment (KostBoh) [reply src/partials/about-us.html:1]: +

Comment (KostBoh) [reply src/partials/about-us.html:1]: +

File: src/partials/about-us.html (hunk 15 above)
Comment (KostBoh) [reply src/partials/about-us.html:3]: +

File: src/css/about-us.css (hunk 13 above)
Comment (KostBoh) [reply src/css/about-us.css:193]: +

File: src/partials/about-us.html (hunk 16 above)
Comment (KostBoh) [reply src/partials/about-us.html:33]: +

File: src/partials/about-us.html (hunk 17 above)
Comment (KostBoh) [reply src/partials/about-us.html:30]: +

File: src/css/about-us.css (hunk 18 above)
Comment (KostBoh) [reply src/css/about-us.css:95]: +

File: src/css/about-us.css (hunk 19 above)
Comment (KostBoh) [reply src/css/about-us.css:97]: +

**Comment by KostBoh** [comment]
I have corrected all comments

File: src/css/about-us.css [hunk 20]
```diff
@@ -0,0 +1,171 @@
+.about-section {
```
Comment (asaulyak) [inline src/css/about-us.css:1]: remove empty block

File: src/css/about-us.css [hunk 21]
```diff
@@ -0,0 +1,171 @@
...
+}
+
+.about-title-container {
+  max-width: 335px;
```
Comment (asaulyak) [inline src/css/about-us.css:5]: remove max-width
it is defined in global container

File: src/css/about-us.css [hunk 22]
```diff
@@ -0,0 +1,171 @@
...
+  margin-bottom: 40px;
+}
+
+@media only screen and (min-width: 768px) {
+  .about-title-container {
+    max-width: 704px;
```
Comment (asaulyak) [inline src/css/about-us.css:11]: remove max-width
it is defined in global container

File: src/css/about-us.css [hunk 23]
```diff
@@ -0,0 +1,171 @@
...
+
+    margin-bottom: 64px;
+  }
+}
+
+@media only screen and (min-width: minwidthpx) {
```
Comment (asaulyak) [inline src/css/about-us.css:17]: remove empty block

File: src/css/about-us.css [hunk 24]
```diff
@@ -0,0 +1,171 @@
...
+  .about-title-container {
+  }
+}
+
+@media only screen and (min-width: 1280px) {
+  .about-title-container {
+    max-width: 832px;
+    /* margin-bottom: 64px; */
```
Comment (asaulyak) [inline src/css/about-us.css:25]: remove

File: src/css/about-us.css [hunk 25]
```diff
@@ -0,0 +1,171 @@
...
+
+@media only screen and (min-width: 768px) {
+  .section-title {
+    font-size: 40px;
+    line-height: 1.2;
+    letter-spacing: -0.02em;
+  }
+}
+
+.accent {
```
Comment (asaulyak) [inline src/css/about-us.css:47]: remove
use global class

File: src/css/about-us.css [hunk 26]
```diff
@@ -0,0 +1,171 @@
...
+  position: relative;
+}
+
+.accent::after {
```
Comment (asaulyak) [inline src/css/about-us.css:51]: remove
use global class

File: src/css/about-us.css [hunk 27]
```diff
@@ -0,0 +1,171 @@
...
+  bottom: 0;
+  content: '';
+  width: calc(100% + 4px);
+  height: 20px;
+  border-radius: 8px;
+  background: var(--soft-green);
+}
+
+.about-section-text {
+  color: rgba(18, 20, 23, 0.5);
```
Comment (asaulyak) [inline src/css/about-us.css:64]: use var for color

File: src/css/about-us.css [hunk 28]
```diff
@@ -0,0 +1,171 @@
...
+
+@media only screen and (min-width: 768px) {
+  .about-section-text {
+    font-size: 18px;
+    line-height: 1.67;
+    max-width: 658px;
+  }
+}
+
+.about-list-container {
```
Comment (asaulyak) [inline src/css/about-us.css:80]: remove this rule

File: src/css/about-us.css [hunk 29]
```diff
@@ -0,0 +1,171 @@
...
+.about-text {
+  width: 100%;
+  height: 48px;
+  flex-shrink: 0;
+  color: var(--black);
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 1.2;
+  letter-spacing: -0.02em;
+  /* padding-left: 32px; */
```
Comment (asaulyak) [inline src/css/about-us.css:143]: remove

File: src/css/about-us.css [hunk 30]
```diff
@@ -0,0 +1,171 @@
...
+  margin-bottom: 16px;
+  display: block;
+}
+
+.about-item {
+  width: 100%;
+  height: 160px;
+  flex-shrink: 0;
+  border-radius: 16px;
+  background: #f3f5f6;
```
Comment (asaulyak) [inline src/css/about-us.css:124]: use color var
use background-color

File: src/partials/about-us.html [hunk 31]
```diff
@@ -1 +1,40 @@
...
+  <div class="container">
+    <h2 class="section-title">About <span class="accent">Us</span></h2>
+    <p class="about-section-text">
+      We are dedicated to providing engaging and effective English language
+      courses in a supportive and stimulating learning environment
+    </p>
+    <div class="about-list-container">
+      <ol class="about-list list">
+        <li class="about-item item">
+          <!-- <p class="about-number">01</p> -->
```
Comment (asaulyak) [inline src/partials/about-us.html:11]: remove

File: src/partials/about-us.html [hunk 32]
```diff
@@ -1 +1,40 @@
...
+          <p class="about-text">Expert, innovative teaching staff</p>
+        </li>
+        <li class="about-item item">
+          <!-- <p class="about-number">02</p> -->
```
Comment (asaulyak) [inline src/partials/about-us.html:15]: remove

File: src/partials/about-us.html [hunk 33]
```diff
@@ -1 +1,40 @@
...
+          <p class="about-text">Wide range of course levels and options</p>
+        </li>
+        <li class="about-item item">
+          <!-- <p class="about-number">03</p> -->
```
Comment (asaulyak) [inline src/partials/about-us.html:19]: remove

File: src/partials/about-us.html [hunk 34]
```diff
@@ -1 +1,40 @@
...
+          <p class="about-text">Multicultural learning environment</p>
+        </li>
+        <li class="about-item item">
+          <!-- <p class="about-number">04</p> -->
```
Comment (asaulyak) [inline src/partials/about-us.html:23]: remove

File: src/css/about-us.css (hunk 22 above)
Comment (KostBoh) [reply src/css/about-us.css:11]: +

File: src/css/about-us.css (hunk 23 above)
Comment (KostBoh) [reply src/css/about-us.css:17]: +

File: src/css/about-us.css (hunk 24 above)
Comment (KostBoh) [reply src/css/about-us.css:25]: +

File: src/partials/about-us.html (hunk 32 above)
Comment (KostBoh) [reply src/partials/about-us.html:15]: +

File: src/partials/about-us.html (hunk 34 above)
Comment (KostBoh) [reply src/partials/about-us.html:23]: +

File: src/partials/about-us.html (hunk 33 above)
Comment (KostBoh) [reply src/partials/about-us.html:19]: +

**Comment by KostBoh** [comment]
made changes

**Review by asaulyak — DISMISSED** [review]

File: src/css/about-us.css (hunk 25 above)
Comment (KostBoh) [reply src/css/about-us.css:35]: +

File: src/css/about-us.css (hunk 26 above)
Comment (KostBoh) [reply src/css/about-us.css:39]: +

File: src/css/about-us.css (hunk 27 above)
Comment (KostBoh) [reply src/css/about-us.css:52]: +

File: src/css/about-us.css (hunk 28 above)
Comment (KostBoh) [reply src/css/about-us.css:69]: +

File: src/css/about-us.css (hunk 29 above)
Comment (KostBoh) [reply src/css/about-us.css:132]: +

**Review by asaulyak — APPROVED** [review]

**Pull request merged**